website/docs/d/cos_bucket.html.markdown

```diff
@@ -44,6 +70,7 @@ Review the argument references that you can specify for your data source.
 - `endpoint_type` - (Optional, String) The type of the endpoint either `public` or `private` or `direct` to be used for the buckets. Default value is `public`.
 - `resource_instance_id` - (Required, String) The ID of the IBM Cloud Object Storage service instance for which you want to create a bucket.
 - `storage_class`- (Required, String)  Storage class of the bucket. Supported values are `standard`, `vault`, `cold`, `smart`.
+- `satellite_location_id` - (Required, String) satellite location id. Provided by end users.
```

Comment:
its optional and make other fields like bucket_type,...son as optioanl as per new changes